server/service/src/sync/translations/contact_form.rs

```diff
@@ -0,0 +1,164 @@
+use repository::{
+    contact_form_row::{ContactFormRow, ContactFormRowRepository},
+    ChangelogRow, ChangelogTableName, StorageConnection, SyncBufferRow,
+};
+
+use crate::sync::translations::{store::StoreTranslation, user::UserTranslation};
+
+use super::{
+    PullTranslateResult, PushTranslateResult, SyncTranslation, ToSyncRecordTranslationType,
+};
+
+// Needs to be added to all_translations()
+#[deny(dead_code)]
+pub(crate) fn boxed() -> Box<dyn SyncTranslation> {
+    Box::new(ContactFormTranslation)
+}
+
+pub(crate) struct ContactFormTranslation;
+
+impl SyncTranslation for ContactFormTranslation {
+    fn table_name(&self) -> &'static str {
+        "contact_form"
+    }
+
+    fn pull_dependencies(&self) -> Vec<&'static str> {
+        vec![UserTranslation.table_name(), StoreTranslation.table_name()]
+    }
+
+    fn try_translate_from_upsert_sync_record(
+        &self,
+        _: &StorageConnection,
+        sync_record: &SyncBufferRow,
+    ) -> Result<PullTranslateResult, anyhow::Error> {
+        Ok(PullTranslateResult::upsert(serde_json::from_str::<
+            ContactFormRow,
+        >(&sync_record.data)?))
+    }
+
+    fn change_log_type(&self) -> Option<ChangelogTableName> {
+        Some(ChangelogTableName::ContactForm)
+    }
+
+    fn should_translate_to_sync_record(
+        &self,
+        row: &ChangelogRow,
+        r#type: &ToSyncRecordTranslationType,
+    ) -> bool {
+        match r#type {
+            ToSyncRecordTranslationType::PullFromOmSupplyCentral => {
```

Comment:
If you only want it to be pushed, you can skip this case...

Reply:
Although You'll need to remove it from the `pull_upsert_records` tests though I think if you do...

Reply:
Yessss, thanks for that James, we we're hoping that we can keep to just be a push! I'll update the test accordingly. But I think this solves our issue right @lache-melvin?

Reply:
I think we tried this John? The issue you were seeing was in the `push` step of pull and push test, it's currently looking for pull records!

We should fix this test (definitely don't want bugs in our tests 😅, especially not around sync!) but I think should be done in a separate issue. I think there might be a bit of clean up involved, shouldn't block/bloat this PR 🙏

Reply:
Yea after thinking about it yesterday we still have to sort that test out. Though I also realised that James left two separate comments haha. I thought they were one.